src/openrct2/world/footpath.c

```diff
@@ -346,7 +346,7 @@ static money32 footpath_place_real(sint32 type, sint32 x, sint32 y, sint32 z, si
         return MONEY32_UNDEFINED;
     }
 
-    if (flags & GAME_COMMAND_FLAG_APPLY)
+    if ((flags & GAME_COMMAND_FLAG_APPLY) && (flags & GAME_COMMAND_FLAG_7) && !(flags & GAME_COMMAND_FLAG_GHOST))
```

Comment:
`GAME_COMMAND_FLAG_7` is set only if you are placing footpath scenery this seems like a mistake. You should interrupt peeps on a footpath place as well since you might be deleteing a paths seat. I'm also going to make an issue saying name `GAME_COMMAND_FLAG_7`